src/Analyzers/CSharp/Analyzers/UseCollectionExpression/UseCollectionExpressionHelpers.cs

```diff
@@ -733,11 +733,20 @@ or ImplicitStackAllocArrayCreationExpressionSyntax
                 .WithPrependedLeadingTrivia(ElasticMarker);
     }
 
+    private static bool HasStructuredTriviaBetweenCloseTokenAndInitializer(InitializerExpressionSyntax initializer)
+    {
+        static bool IsComment(SyntaxTrivia trivia) => trivia.IsKind(SyntaxKind.SingleLineCommentTrivia) || trivia.IsKind(SyntaxKind.MultiLineCommentTrivia);
+        return initializer.OpenBraceToken.GetPreviousToken().TrailingTrivia.Any(IsComment);
```

Comment:
we have a utility method for this already: IsSingleOrMultiLineComment.  I would use that, remove this method, and just inline this code at the callsites.